queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py

```diff
@@ -28,24 +28,25 @@ def consume_nr(business: Business, filing: Filing, nr_num_path='/filing/incorpor
     """Update the nr to a consumed state."""
     try:
         nr_num = get_str(filing.filing_json, nr_num_path)
-        bootstrap = RegistrationBootstrap.find_by_identifier(filing.temp_reg)
-        namex_svc_url = current_app.config.get('NAMEX_API')
-        token = AccountService.get_bearer_token()
+        if nr_num:
```

Comment:
What does it mean if the NR Num is not found?

Reply:
Under what conditions would this happen?

Reply:
for IA this function is called in both cases (numbered and named), so for numbered we have to skip this call

Reply:
Thanks.
Please add code comment explaining this.